pyproject.toml

```diff
@@ -90,4 +89,11 @@ commands =
     rm -rf {toxinidir}/dist
     cp --no-target-directory --recursive {distdir} {toxinidir}/dist; # copy the wheel built by tox-wheel
     {envpython} -m pep517.build --source --out-dir={toxinidir}/dist {toxinidir}
+
+[testenv:coveralls]
+passenv = GITHUB_*
+deps =
+  coveralls
+  coverage>=5.3
+commands = coveralls
```

Comment:
@adiroiban I used coveralls because that's what I'm familliar with and use in https://github.com/PyCQA/modernize

Reply:
I am ok with using coveralls .... a long time ago, I was using coveralls , but it was failing so I switched to codecov.

I think that it would be interesting to run both coveralls and codecov in parallel ... but to be done in a separate PR.

Maybe coveralls should be left for a separate PR and for this PR just focus on adding the basic GitHub actions matrix.